tuf/api/metadata.py

```diff
@@ -680,4 +792,8 @@ def to_dict(self) -> Dict[str, Any]:
     # Modification.
     def update(self, filename: str, fileinfo: Mapping[str, Any]) -> None:
```

Comment:
Same question here, is this a useful function vs caller modifying targets.signed.targets directly

Reply:
It will be the same I suppose, but I hope we will use this function to make some validation before doing the assignment.

Reply:
good point, thanks